ga4gh/cli.py

```diff
@@ -192,6 +192,18 @@ def getAllReadGroupSets(self):
             for readGroupSet in iterator:
                 yield readGroupSet
 
+    def getAllReadGroups(self):
+        """
+        Get all read groups in a read group set
+        """
+        for dataset in self.getAllDatasets():
+            iterator = self._client.searchReadGroupSets(
+                datasetId=dataset.id)
+            for readGroupSet in iterator:
+                rgs = self._client.getReadGroupSet(readGroupSet.id)
+                for rg in rgs.readGroups:
+                    yield rg.id
```

Comment:
We should yield the readgroup object here rather than the ID to keep symmetry with the other methods.